GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationView.swift

```diff
@@ -159,9 +169,9 @@ extension SecureConversations {
         }
 
         private func renderProps() {
-            header.title = props.style.headerTitle
-            header.backButton.tap = props.backButtonTap.execute
-            header.closeButton.tap = props.closeButtonTap.execute
+            header.props.title = props.style.headerTitle
+            header.backButton.props.tap = .init { [weak self] in self?.props.backButtonTap.execute() }
```

Comment:
How about forming props for back button and close button all together with ConfirmationView.Props at once?

Reply:
fixed - 997017ba6c73b2161cee6aef5164eb16e074d9d6